FROST-Server.Core/src/main/java/de/fraunhofer/iosb/ilt/frostserver/service/Service.java

```diff
@@ -309,7 +309,8 @@ private ServiceResponse executeGetCapabilities(ServiceRequest request) {
             LOGGER.error("Formatter not available.", ex);
             return errorResponse(response, 500, "Failed to instantiate formatter");
         }
-        response.setResultFormatted(formatter.format(null, null, result, settings.getQueryDefaults().useAbsoluteNavigationLinks()));
+        response.setResultFormatted(
+                formatter.format(null, null, result, settings.getQueryDefaults().useAbsoluteNavigationLinks()));
```

Comment:
Please avoid edits that only change formatting.

Reply:
This is fixed.